### DISCUSSION TITLE
How to use gradient-heading now that GradientHeading component is deprecated?

### DISCUSSION THREAD

**Discussion by AgarwalPragy** [skeletonlabs/skeleton#795] [Q&A] [answered]
[Docs mention](https://www.skeleton.dev/components/gradient-headings) the following

```css
.gradient-heading {
    @apply bg-gradient-to-r from-primary-500 to-bg-secondary-500 bg-clip text-transparent box-decoration-clone;
}
```

However, doing that gives the following error

```markdown
The `to-bg-secondary-500` class does not exist. If `to-bg-secondary-500` is a custom class, make sure it is defined within a `@layer` directive.
```

Putting the code in the layer directive doesn't help either.

**Comment by endigo9740** [comment, marked as answer]
@AgarwalPragy Looks like a typo in the docs. Change:

`to-bg-secondary-500` -> `to-secondary-500`

**Reply by AgarwalPragy** [reply]
Ah.. I see. Thanks! :)

additionally, `bg-clip` -> `bg-clip-text`
